app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -108,7 +109,8 @@ public static void handleNewBgReading(BgReading bgReading, String operation_type
                     context.sendBroadcast(updateIntent);
                 }
 
-                if (AppWidgetManager.getInstance(context).getAppWidgetIds(new ComponentName(context, xDripWidget.class)).length > 0) {
+                Context appContext = xdrip.getAppContext();//KS
```

Comment:
I wasn't sure about this bit, is it needed? what was the reason for this change?